pallets/phala-world/src/traits.rs

```diff
@@ -69,6 +69,10 @@ pub enum PartRarityType {
 pub enum PartSizeType {
 	MA,
 	MB,
+	MC,
```

Comment:
Not sure if the enum is already in used in the storage. If so, please only append to the enum to keep compatibility.

Reply:
Not yet. For now, no Shell NFT has been hatched. This mistake found in local testnet when testing hatching shell NFT with real-world data.